Docker/SolrServer/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM solr:9.6.1
-MAINTAINER dkd Internet Service GmbH <info@dkd.de>
-ENV TERM linux
+LABEL maintainer="dkd Internet Service GmbH info@dkd.de"
```

Comment:
The recommended replacement for `MAINTAINER` differs from this solution, see
https://docs.docker.com/reference/dockerfile/#maintainer-deprecated

IMHO we should adjust this and use `LABEL org.opencontainers.image.authors`